marimo/_ast/app.py

```diff
@@ -398,6 +399,29 @@ async def embed(self) -> AppEmbedResult:
             )
 
 
+class AppMeta:
+    """
+    Metadata about the app.
+
+    This is used to store metadata about the app
+    that is not part of the app's code or state.
+    """
+
+    def __init__(self) -> None:
+        self.user_config = load_config()
+        self.apply_theme()
```

Comment:
Should we move this `apply_theme` to somewhere else, like sessions or runtime?

Reply:
we can probably make it opt-in for now, so you can remove this. 

and we can have people do:
```
import marimo as mo
mo.apply_theme_to_third_party()
```